ddtrace/appsec/_iast/_handlers.py

```diff
@@ -403,3 +405,65 @@ def _on_set_request_tags_iast(request, span, flask_config):
             OriginType.PARAMETER,
             override_pyobject_tainted=True,
         )
+
+
+def _on_django_finalize_response_pre(ctx, after_request_tags, request, response):
+    from .._asm_request_context import is_stacktrace_reported
```

Comment:
Can't we have this import at top-level?

Reply:
No, circular import.

Reply:
PS: removed the local imports now that it has moved to the IAST context.